frontend/src/pages/Logs/index.tsx

```diff
@@ -18,16 +18,23 @@ import { SET_SEARCH_QUERY_STRING } from 'types/actions/logs';
 import SpaceContainer from './styles';
 
 function Logs({ getLogsFields }: LogsProps): JSX.Element {
-	const urlQuery = useUrlQuery();
+	const mountedRef = useRef<boolean>(false);
 
+	const urlQuery = useUrlQuery();
 	const dispatch = useDispatch();
 
 	useEffect(() => {
-		dispatch({
-			type: SET_SEARCH_QUERY_STRING,
-			payload: urlQuery.get('q'),
-		});
-	}, [dispatch, urlQuery]);
+		mountedRef.current = true;
+	}, [mountedRef]);
+
+	useEffect(() => {
+		if (!mountedRef.current) {
+			dispatch({
+				type: SET_SEARCH_QUERY_STRING,
+				payload: urlQuery.get('q'),
+			});
+		}
+	}, [dispatch, mountedRef, urlQuery]);
```

Comment:
This change was added considering that this was just used to set query string once we refresh or redirect to the URL with query

Reply:
✌️